tests/system/feature_store/test_feature_store.py

```diff
@@ -4813,6 +4813,77 @@ def test_merge_different_number_of_entities(self):
         ).to_dataframe()
         assert_frame_equal(expected_all, df, check_dtype=False)
 
+    @staticmethod
+    def _get_sorted_df(df: pd.DataFrame, sort_column: str):
+        result = (
+            df.reindex(sorted(df.columns), axis=1)
+            .sort_values(by=sort_column)
+            .reset_index(drop=True)
+        )
+        return result
```

Comment:
```suggestion
    def _sort_df(df: pd.DataFrame, sort_column: str):
        return (
            df.reindex(sorted(df.columns), axis=1)
            .sort_values(by=sort_column)
            .reset_index(drop=True)
        )
```

Reply:
Please notice that my suggestion also renames the function (looks like you only inlined `result`).

Reply:
fixed, tnx